notebooks/bonus-unit1/bonus-unit1.ipynb

```diff
@@ -582,6 +582,8 @@
       "source": [
         "1. In step 1, choose your model repository which is the model id (in my case ThomasSimonini/ppo-Huggy).\n",
         "\n",
+        "👉 If you have any racing wheels/flight sim components connected to your computer be sure to disconnect them or you may have trouble selecting your model from the dropdown.",
```

Comment:
Can you add a space after this otherwise 2. is glued after this text:

![Uploading image.png…]()